src/main/java/com/github/lyrric/service/SecKillRunnable.java

```diff
@@ -0,0 +1,76 @@
+package com.github.lyrric.service;
+
+import com.github.lyrric.conf.Config;
+import com.github.lyrric.model.BusinessException;
+import org.apache.http.conn.ConnectTimeoutException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.net.SocketTimeoutException;
+
+/**
+ * @author wangxiaodong
+ */
+public class SecKillRunnable implements Runnable{
+
+    private final Logger logger = LogManager.getLogger(SecKillService.class);
+    /**
+     * 是否刷新st
+     */
+    private boolean resetSt;
+    /**
+     * httpService
+     */
+    private HttpService httpService;
+    /**
+     * 疫苗id
+     */
+    private Integer vaccineId;
+    /**
+     * 开始时间
+     */
+    private long startDate;
+
+    public SecKillRunnable(boolean resetSt, HttpService httpService, Integer vaccineId, long startDate) {
+        this.resetSt = resetSt;
+        this.httpService = httpService;
+        this.vaccineId = vaccineId;
+        this.startDate = startDate;
+    }
+
+    @Override
+    public void run() {
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 15. By comparison, 99% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.